src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser.md

```diff
@@ -3,22 +3,21 @@ title: "Browser"
-| [browser.newPage([options])](/javascript-api/xk6-browser/browser/newpage)                      | Creates a new Page in a new BrowserContext and returns the page.                                                |
-| 🚧 [browser.on('disconnected')](/javascript-api/xk6-browser/browser/on)                       | Detects events from the browser application.                                                                    |
-| ❌ [browser.startTracing()](/javascript-api/xk6-browser/browser/starttracing)                  | Starts Chromium Tracing for debugging a Page.                                                                   |
-| ❌ [browser.stopTracing()](/javascript-api/xk6-browser/browser/stoptracing)                    | Stops the tracing created by the browser.startTracing() method.                                                 |
-| [browser.version()](/javascript-api/xk6-browser/browser/version)                               | Returns the browser application's version.                                                                      |
+| Method                                                                                    | Description                                                                                                                                           |
+|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [browser.close()](/javascript-api/xk6-browser/browser/close)                              | Closes the browser and all of its pages (if any were opened).                                                                                         |
+| [browser.contexts()](/javascript-api/xk6-browser/browser/contexts)                        | Allows you to access all open [BrowserContexts](/javascript-api/xk6-browser/browsercontext/)s.                                                        |
+| <BWIPT id="453"/> [browser.isConnected](/javascript-api/xk6-browser/browser/isconnected)  | Indicates whether the [CDP](https://chromedevtools.github.io/devtools-protocol/) connection to the browser process is active or not.                  |
+| <BWIPT/> [browser.newContext([options])](/javascript-api/xk6-browser/browser/newcontext/) | Creates and returns a new [BrowserContexts](/javascript-api/xk6-browser/browsercontext/).                                                             |
+| <BWIPT/> [browser.newPage([options])](/javascript-api/xk6-browser/browser/newpage)        | Creates a new [Page](/javascript-api/xk6-browser/page/) in a new [BrowserContexts](/javascript-api/xk6-browser/browsercontext/) and returns the page. |
```

Comment:
I wonder if we should create issues to link these WIP icons to, and in general, always have a link in these cases.

Otherwise it's not clear what exactly isn't working, or what's missing for it to be fully working. It also makes it visible and plannable for us, as I'm currently not sure why these are WIP.

This is not a blocker for this PR, but WDYT?

Reply:
Yep, that is a little confusing. So the reason I've let them as WIP is because a few of the options are still WIP. Happy to create an issue though that encapsulates the issues with the options.

Reply:
Resolved in: 92c839a3765d609c2b43322595652554695f5381